test/functional/cmdLineTests/criu/criu_nonPortable.xml

```diff
@@ -275,7 +275,7 @@
     <output type="success" caseSensitive="no" regex="no">Killed</output>
     <output type="failure" caseSensitive="yes" regex="no">failed properties test</output>
     <output type="required" caseSensitive="yes" regex="no">Pre-checkpoint</output>
-    <output type="success" caseSensitive="yes" regex="no">Post-checkpoint</output>
+    <output type="required" caseSensitive="yes" regex="no">Post-checkpoint</output>
```

Comment:
cant have required conditions because of intermittent criu failures. All failure cases need to be explicitly listed in the test case

Reply:
Updated the test with explicit exceptions and error messages.